lang/cs/Org.Apache.REEF.Driver/Bridge/Events/EvaluatorRequestor.cs

```diff
@@ -33,9 +34,10 @@ namespace Org.Apache.REEF.Driver.Bridge.Events
     [DataContract]
     internal class EvaluatorRequestor : IEvaluatorRequestor
     {
+        internal const char BatchIdxSeparator = '_';
         private static readonly Logger LOGGER = Logger.GetLogger(typeof(EvaluatorRequestor));
-
-        private static Dictionary<string, IEvaluatorDescriptor> _evaluators;
+        private static readonly Lazy<IDictionary<string, IEvaluatorDescriptor>> EvaluatorsInstantiator = 
```

Comment:
Would it be much simpler to create the Dictionary at this line itself? 
private static readonly IDictionary<string, IEvaluatorDescriptor> EvaluatorsInstantiator = new Dictionary<string, IEvaluatorDescriptor>()
We need this dictionary anyway and creating an empty dictionary at static time has no any risk.